server/src/test/resources/csars/yaml/valid/lamp-csar/apache_web_server/webserver_install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo apt-get install apache2 -y
```

Comment:
I Really dont like the idea of having duplicate code here, maybe consider putting both variants in the same directory. Just change the name of the `yaml` file (assuming nothing else has changed)